snap/hooks/pre-refresh

```diff
@@ -3,4 +3,4 @@
 # save this revision for when we run again in the post-refresh
 snapctl set lastrev="$SNAP_REVISION"
 
-snapctl set release="geneva"
```

Comment:
github shows this file lost its execute bit, but the file has a shebang, so it should probably be executable.

Run `chmod +x <this file>`

Reply:
I have made the file an executable